modules/asm/main.tf

```diff
@@ -33,6 +33,6 @@ module "asm_install" {
   service_account_key_file = var.service_account_key_file
 
 
-  kubectl_create_command  = "${path.module}/scripts/install_asm.sh ${var.project_id} ${var.cluster_name} ${var.location} ${var.asm_version}"
+  kubectl_create_command  = "${path.module}/scripts/install_asm.sh ${var.project_id} ${var.cluster_name} ${var.location} ${var.asm_version} ${var.managed}"
```

Comment:
This could cause a delete recreate operation since we are changing the command and would need a breaking release. An alternative could be to keep the command as is and switch based on `${var.managed}` and default managed to false in the script.

` kubectl_create_command = var.managed ? "${local.kubectl_create_command_base} ${var.managed} : local.kubectl_create_command_base`

Reply:
Agreed, let's set a default and keep this backwards-compatible.

Reply:
Makes totally sense! Submitting the updated patch